forms-flow-components/src/components/CustomComponents/FormBuilderModal.tsx

```diff
@@ -136,7 +149,7 @@ export const FormBuilderModal: React.FC<BuildFormModalProps> = React.memo(
              id="anonymouseCheckbox"
              label={t("Allow adding multiple pages form in this form")}
              checked={checked}
-             onChange={setChecked}
+             onChange={setChecked} //TBD: need to remove and add to setValues 
```

Comment:
if name and description is moved under variable 'values' can we move this one also ? In that case please make sure that it is an optional item to pass as this is used as a DUPLICATE FORM modal also where we are not passing this checkbox.  @shuhaib-aot